src/test/java/TestNGTests/TestGet.java

```diff
@@ -0,0 +1,32 @@
+package TestNGTests;
+
+import java.io.IOException;
+import APICalls.GetCall;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import utility.PropertyManager;
+
+public class TestGet extends GetCall {
```

Comment:
please check common comments that were added in assertJ and restAssured packages